app/models/host_status/compliance_status.rb

```diff
@@ -0,0 +1,46 @@
+module HostStatus
```

Comment:
this should be also under `ForemanOpenscap` module and hence in app/models/foreman_openscap/host_status (host status might not be needed, but leaving this up to you)